nss-day-cohort-30/bangazon-site-suave-salmon
# Happy

Fixes:
-PaymentType model, view and controller update to display an error message if the expiration date is not valid. Also, _Layout page update to display the message. 
-

Changes proposed in this request & reasons behind organizational or architectural decisions:
-
-

Steps to test:
- need to install new migration in order to test
- steps to uninstall and reinstall is on our slack channel
-
-

System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):
-
-

Commit message:
-alert message box displays if the expiration date is not correct"
-updated paymentType model and view. view contains the section fo r expiration date. updated paymentType controller to create method to prevent us ers from putting dates in the past"
--




Link to feature ticket:
-http://localhost:5000/PaymentTypes/Create

@suave-salmon
